index.bs

```diff
@@ -3245,17 +3237,143 @@ A [=remote end=] has a <dfn>before request sent map</dfn> which is initially an
 empty map.  It's used to track the network events for which a
 <code>network.beforeRequestSent</code> event has already been sent.
 
+### Network Intercepts ### {#network-intercepts}
+
+A <dfn>network intercept</dfn> is a mechanism to allow remote ends to intercept
+and modify network requests and responses.
+
+A [=BiDi session=] has an <dfn>intercept map</dfn> which is initially an
```

Comment:
I suppose it's from "Intercept" to "urlPatterns" and "phases".

Can we note this right here? Otherwise the reader needs to go all the way to https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/429.html#command-network-addIntercept in order to figure it out.